homeassistant/components/google_assistant/trait.py

```diff
@@ -2385,18 +2401,56 @@ def supported(cls, domain, features, device_class, _):
     def sync_attributes(self):
         """Return attributes for a sync request."""
         device_class = self.state.attributes.get(ATTR_DEVICE_CLASS)
-        if (data := self.sensor_types.get(device_class)) is not None:
+        data = self.sensor_types.get(device_class)
+
+        if data[0] == self.sensor_types.get(sensor.SensorDeviceClass.AQI)[0]:
             return {
-                "sensorStatesSupported": {
-                    "name": data[0],
-                    "numericCapabilities": {"rawValueUnit": data[1]},
-                }
+                "sensorStatesSupported": [
+                    {
+                        "name": data[0],
+                        "numericCapabilities": {"rawValueUnit": data[1]},
+                        "descriptiveCapabilities": {
+                            "availableStates": [
+                                "healthy",
+                                "moderate",
+                                "unhealthy for sensitive groups",
+                                "unhealthy",
+                                "very unhealthy",
+                                "hazardous",
+                                "unknown",
+                            ],
+                        },
+                    }
+                ]
+            }
+        if data is not None:
+            return {
+                "sensorStatesSupported": [
+                    {
+                        "name": data[0],
+                        "numericCapabilities": {"rawValueUnit": data[1]},
+                    }
+                ]
             }
 
     def query_attributes(self):
         """Return the attributes of this trait for this entity."""
         device_class = self.state.attributes.get(ATTR_DEVICE_CLASS)
-        if (data := self.sensor_types.get(device_class)) is not None:
+        data = self.sensor_types.get(device_class)
+
+        if data[0] == self.sensor_types.get(sensor.SensorDeviceClass.AQI)[0]:
```

Comment:
This is just weird. Just check the device_class directly here.

```suggestion
        if device_class == sensor.SensorDeviceClass.AQI:
```